src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withStyles } from "@material-ui/styles";
+
+import SampleServices from "../../../../assets/thirdPartyServices";
+import { useStyles } from "./styles";
+import { serviceActions } from "../../../../Redux/actionCreators";
+import { APIEndpoints } from "../../../../config/APIEndpoints";
+import Session from "../../../../utility/constants/Session";
+
+class ThirdPartyAIService extends Component {
+  state = {
+    grpcResponse: undefined,
+    AIServiceCustomComponent: undefined,
+    serviceSpecJSON: undefined,
+    protoSpec: undefined,
+  };
+
+  sampleServices = new SampleServices();
+
+  componentDidUpdate = () => {
+    const { org_id, service_id } = this.props;
+    if (org_id && service_id && !this.state.AIServiceCustomComponent) {
+      this.fetchServiceSpec(org_id, service_id);
+      const AIServiceCustomComponent = this.sampleServices.getComponent(org_id, service_id);
+      this.setState({ AIServiceCustomComponent });
+    }
+  };
+
+  fetchServiceSpec = async (org_id, service_id) => {
+    let servicebufURL = `${APIEndpoints.SERVICE_BUF.endpoint}/${org_id}/${service_id}`;
```

Comment:
`const`?. I would suggest to start with constant and then convert to let when needed.

Reply:
[done](https://github.com/singnet/refactored-dapp/pull/13/commits/bb0296bb37189c4bf226d4d7fdccf7b13ca205ab)